test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/BatchTests.cs

```diff
@@ -42,5 +88,35 @@ public async Task WhenSubmittingABatch_GivenANonBundleResource_ThenBadRequestIsR
 
             Assert.Equal(HttpStatusCode.BadRequest, ex.StatusCode);
         }
+
+        private void ValidateResourceStatusCode(Bundle resource)
```

Comment:
might want to rename the method name to ValidateResourceOutput since we are validating both status and operationoutcome in case of failures as well?